aspnetcore/blazor/components.md

```diff
@@ -511,6 +511,126 @@ Use `EventCallback` and `EventCallback<T>` for event handling and binding compon
 
 Prefer the strongly typed `EventCallback<T>` over `EventCallback`. `EventCallback<T>` provides better error feedback to users of the component. Similar to other UI event handlers, specifying the event parameter is optional. Use `EventCallback` when there's no value passed to the callback.
 
+## Chained bind
+
+A common scenario is chaining a data-bound parameter to a page element in the component's output. This scenario is called a *chained bind* because multiple levels of binding occur simultaneously.
+
+Chained bind can't be implemented with `@bind` syntax in the page's element. Chained bind is created using the property with an event handler. A parent component can use `@bind` syntax with the parameter.
+
+The following `PasswordField` component (*PasswordField.razor*):
+
+* Sets an `<input>` element's value to a `Password` property.
+* Exposes changes of the `Password` property to a parent component with an [EventCallback](#eventcallback).
+
+```cshtml
```

Comment:
Hopefully this will eventually be migrated to code file.
i can do that post GA

Reply:
I added a bullet point for this to the existing issue WRT what the sample will cover and what it won't. There was no point to doing the extra work for snippets at this time with so much churn and without engineering identifying which examples they want in the sample app and which ones they just want in-text. I used snippets when I needed highlighting. Otherwise, it wasn't worth the cost during preview. 🏃

https://github.com/aspnet/AspNetCore.Docs/issues/14216